icu4j/main/core/src/test/java/com/ibm/icu/dev/test/segmenter/RuleBasedSegmenterTest.java

```diff
@@ -27,6 +27,14 @@ public void testRules() {
             "Kühlschränke kühlen Getränke",
             "[a-z]+;",
             Arrays.asList("K", "ü", "hlschr", "ä", "nke", " ", "k", "ü", "hlen", " ", "G", "etr", "ä", "nke")},
+        {"ASCII upper- and lowercase a-z",
+            "Kühlschränke kühlen Getränke",
+            "[A-Za-z]+;",
+            Arrays.asList("K", "ü", "hlschr", "ä", "nke", " ", "k", "ü", "hlen", " ", "Getr", "ä", "nke")},
+        {"ASCII upper- and lowercase a-z and some letters with diaeresis",
+            "Kühlschränke kühlen Getränke",
+            "[A-Za-z{ü}{ä}]+;",
```

Comment:
nit: unless the accented characters are encoded in NFD, they don't need the curly braces

Reply:
Done.